OpenEdXMobile/res/layout/view_register_spinner.xml

```diff
@@ -3,32 +3,30 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:orientation="vertical"
-    android:paddingTop="4dp"
-    android:paddingBottom="2dp">
+    android:paddingTop="@dimen/registration_field_vertical_padding"
+    android:paddingBottom="@dimen/registration_field_vertical_padding"
+    android:orientation="vertical">
 
     <org.edx.mobile.module.registration.view.RegistrationOptionSpinner
         android:id="@+id/input_spinner"
-        style="@style/spinner"
+        style="@style/Base.Widget.AppCompat.Spinner.Underlined"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:textColorHint="@color/hint_grey_text"
-        android:paddingLeft="5dp"
-        android:paddingStart="5dp"
-        android:paddingRight="10dp"
-        android:paddingEnd="10dp"
+        android:textColorHint="@color/grey_text"
         android:focusable="true"
+        android:paddingStart="@dimen/registration_spinner_horizontal_padding"
```

Comment:
Nevermind. I figured it out. Padding **in this case** doesn't affect RTL since we don't care about symmetry.